relays/tasmota/firmware.lua

```diff
@@ -0,0 +1,77 @@
+json = require('json')
+connection = http.client({ timeout = 10 })
+local config = require('enapter.ucm.config')
+
+-- The addres is the local IP adres of the tasmota device
+-- The password is optional it is only needed in case the web-admin pasword is set on the tasmota device,
+-- be aware that this is only a very thin extra layer of security and you should not purely rely on that.
+ADDRESS = 'address'
+PASSWORD = 'password'
+
+function main()
+  config.init({
+    [PASSWORD] = { type = 'string', required = false },
+    [ADDRESS] = { type = 'string', required = true, default = '127.0.0.1' },
+  })
+  enapter.register_command_handler('turn_on', function()
+    return control_device('Power%20On')
+  end)
+  enapter.register_command_handler('turn_off', function()
+    return control_device('Power%20Off')
+  end)
+  enapter.register_command_handler('tasmota_command', tasmota_command)
+  scheduler.add(30000, send_properties)
+  scheduler.add(1000, send_telemetry)
+end
+
+function tasmota_command(args)
```

Comment:
I think here the first argument (context, `ctx`) is missing: https://developers.enapter.com/docs/reference/ucm/enapter#enapterregister_command_handler

There are a couple of options on how to fix this.

First, you can `log` into a command execution context, so that the log will be attached to the command:

```lua
function tasmota_command(ctx, args)
  ctx.log('command arrived ' .. args.command)
  return control_device(args.command)
end
```

Or if you do not want to use `ctx`, then either call it `_` or `_ctx` to make it explicit (as per linter recommendation):

```lua
function tasmota_command(_ctx, args)
  enapter.log('command arrived ' .. args['command'])
  return control_device(args['command'])
end